tests/store/model_registry/test_file_store.py

```diff
@@ -749,15 +771,109 @@ def search_versions(filter_string):
         name=mv1.name, version=mv1.version, description="Online prediction model!"
     )
 
-    mvds = store.search_model_versions(f"run_id = '{run_id_1}'")
-    assert len(mvds) == 1
+    mvds = store.search_model_versions("run_id = '%s'" % run_id_1, max_results=10)
+    assert 1 == len(mvds)
     assert isinstance(mvds[0], ModelVersion)
     assert mvds[0].current_stage == "Production"
     assert mvds[0].run_id == run_id_1
     assert mvds[0].source == "A/B"
     assert mvds[0].description == "Online prediction model!"
 
 
+def test_search_model_versions_order_by_simple(store):
+    # create some model versions
+    prefix = "test_for_search_MV_"
```

Comment:
Do we need this prefix?

Reply:
Not really, I just use it as a convention as other tests :) Do you want me to remove it?

Reply:
Let's remove it!